cli/main.rs

```diff
@@ -914,15 +913,15 @@ pub fn main() {
         eprintln!("{}", e);
         std::process::exit(1);
       }
-      return;
+      std::process::exit(0);
     }
     DenoSubcommand::Types => {
       let types = get_types(flags.unstable);
       if let Err(e) = write_to_stdout_ignore_sigpipe(types.as_bytes()) {
         eprintln!("{}", e);
         std::process::exit(1);
       }
-      return;
+      std::process::exit(0);
```

Comment:
this fine for now, but ideally we would architect it so that `exit` is not called deeper inside the system

Reply:
Agreed, I'm still looking for proper patterns